app/services/invoices/add_on_service.rb

```diff
@@ -61,8 +61,8 @@ def compute_amounts(invoice)
 
     def create_add_on_fee(invoice)
       fee_result = Fees::AddOnService
-        .new(invoice: invoice, applied_add_on: applied_add_on).create
-      raise(fee_result.throw_error) unless fee_result.success?
+        .new(invoice:, applied_add_on:).create
```

Comment:
New Ruby 3.1 syntax. Rubocop enforces it.

I don't know if we want to keep it like that, let me know :smile: